src/SdssShape.cc

```diff
@@ -871,8 +857,8 @@ FluxResult SdssShapeAlgorithm::computeFixedMomentsFlux(ImageT const &image,
                                       .str());
         }
         double var = ImageAdaptor<ImageT>().getVariance(image, ix, iy);
-        double i0Err = std::sqrt(var / wArea);
-        result.instFluxErr = i0Err * 2 * wArea;
+        // i0Err = std::sqrt(var / wArea); instFluxErr = 2 * wArea * i0Err
```

Comment:
Remove the commented-out code, but add a real comment to show where your final formula came from.